_posts/release/2018-07-18-accumulo-1.9.3.md

```diff
@@ -4,8 +4,7 @@ draft: true
 ---
 
 Apache Accumulo 1.9.3 contains bug fixes for Write Ahead Logs and compaction.
-Users of any previous version of 1.8 or 1.9 are encouraged to upgrade
-~~immediately or specify recommended timeline~~.
+Users of any previous version of 1.8 or 1.9 are encouraged to upgrade.
```

Comment:
1.9.2 users are encouraged to upgrade. Users of previous versions should have already upgraded, due to critical bugs in earlier versions. I wonder if we can tweak this wording to remind them of the critical nature of the previous bugs (perhaps with gentle shaming :smile:).